cwd/app.js

```diff
@@ -299,6 +299,8 @@
 
   };
 
+  prevGauge = 0;
```

Comment:
Could this be defined in the same place as `gaugeIndex`, given that they are linked in terms of behavior?  I would also consider making it an object, like:
`index = { current: 3, previous: 2 }`
Definitely not a requirement, though.

Reply:
Turns out `prevGauge` wasn't even needed.